### ISSUE TITLE
How to Grab Settings from a Premium II Server?

### ISSUE THREAD

**Issue by monipoka** [vivinano/MudaeAutoBot#100]
The bot was running fine until my server upgraded to premium II. Now, I am unable to grab the settings. I have restarted my computer, triggered the $settings command, and I tried to manually add them by creating a file within the "channeldata" folder. Is there a way to fix this? 

Thank you!! :)

**Comment by vivinano** [comment]
So basically what is happening it that because u have an existing file it is not doing another file grab you need to delete the file.txt associated with the channel and it should pull a new one

The reasoning for the channeldata was to keep persistent data to avoid issues of not finding settings due to high traffic so if it finds a existing file it will use that and not update it

**Comment by monipoka** [comment]
I deleted the old txt file and reran the bot. Unfortunately, I am still encountering the same error. After about 10 minutes of loading, I get a notification saying that the settings could not be found and to trigger the $settings command.

**Comment by vivinano** [comment]
You could manually enter it cause all it is basically doing

Is taking the $settings and pasting it in a txt file 

So basically take the $settings text and paste it inside while removing the emojis

**Comment by monipoka** [comment]
I tried that. 

I'm really confused with what's happening. I thought it might have been a server premium error, but the bot isn't recognizing settings from any server. 

After that, I deleted the bot and discum, reinstalled, tried multiple servers, manually added settings, and I encounter the same problem. 

Thank you for your response though!

**Comment by vivinano** [comment]
Hmm and all your setting are okay ? Cause my server has premium so it was okay with that one. So is the issue there is not a new one being created right now ? 

Like If you manually create one like copy paste one it still doesn’t work ?

**Comment by monipoka** [comment]
Yeah, I copied and pasted the settings into a txt file with the server id as the name and still no luck.

**Comment by vivinano** [comment]
Show me the channel id and the .txt

**Comment by vivinano** [comment]
![image](https://user-images.githubusercontent.com/33008397/134096864-e3f28477-0798-4456-8a94-2b61c2874d4e.png)

this is what the channeldata is suppose to look like basically where is says server not premium would be replaced with server preimum II bascially it it removes all emoji's from the $settings because emojis will not be parsed by the code

**Comment by vivinano** [comment]
Haven't heard a response. Will be closing issue

TLDR: $settings were changed and bot needed to grab fresh settings instance. Emojis are present when a server is premium and must be replaced with _ or a `empty` as emojis are unable to be parsed.
